pages/Product/components/Reviews/components/List/components/Review/components/Info/components/Author/index.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+
+/**
+ * Review Author Component
+ * @param {string} author The author's name
+ * @returns {null|JSX}
+ */
+const Author = ({ author }) => {
+  if (!author) {
+    return null;
+  }
+
+  return <I18n.Text string="reviews.author" params={[author]} />;
```

Comment:
Please give the parameter a name by passing an object instead of an array: `params={{ author }}`. **Attention:** You need to adjust the locale files as well!

Reply:
Done.